### ISSUE TITLE
pip version outdated/broken

### ISSUE THREAD

**Issue by Joool** [crytic/evm_cfg_builder#24]
The version published on pip seems to be outdated (the version tag is identically though). When I install the current version published on pip I get the following message when trying to export the cfg as a dot file:

```
Traceback (most recent call last):
  File "/Users//.pyenv/versions/evm_toolkit/bin/evm-cfg-builder", line 11, i
n <module>
    sys.exit(main())
  File "/Users//.pyenv/versions/3.6.4/envs/evm_toolkit/lib/python3.6/site-pa
ckages/evm_cfg_builder/__main__.py", line 63, in main
    output_to_dot(args.dot_directory, args.filename, cfg)
  File "/Users//.pyenv/versions/3.6.4/envs/evm_toolkit/lib/python3.6/site-pa
ckages/evm_cfg_builder/__main__.py", line 21, in output_to_dot
    function.output_to_dot(filename)
  File "/Users//.pyenv/versions/3.6.4/envs/evm_toolkit/lib/python3.6/site-pa
ckages/evm_cfg_builder/cfg/function.py", line 146, in output_to_dot
    self.output_dispatcher_to_dot(base_filename)
  File "/Users//.pyenv/versions/3.6.4/envs/evm_toolkit/lib/python3.6/site-pa
ckages/evm_cfg_builder/cfg/function.py", line 181, in output_dispatcher_to_dot
    if self.key in basic_block.incoming_basic_blocks_as_dict:
AttributeError: 'BasicBlock' object has no attribute 'incoming_basic_blocks_as_dict'
```
Which was an error fixed in commit 0e1009f000277b2929b86c1925d387a24766edf9. However, the fix does not seem to be propagated to pip yet.

**Comment by montyly** [comment]
Hi @Joool 

Thank you for reporting the issue. The pip version is indeed outdated, we will make a new release soon

**Comment by montyly** [comment]
Hi @Joool,

We released [0.3.0](https://github.com/crytic/evm_cfg_builder/releases/tag/0.3.0) on pip. you should not have this issue anymore.
